client/app/components/message/Message.jsx

```diff
@@ -1,15 +1,20 @@
 'use strict';
 
-var React = require('react');
+var React = require('tux/React');
 var MessageForm = require('./MessageForm.jsx');
 
-var Message = React.createClass({
+var Message = React.createMutableClass({
   propTypes: {
     message: React.PropTypes.object.isRequired,
     deleteMessage: React.PropTypes.func,
```

Comment:
With the ownerProps will the deleteMessage and updateMessage functions be on a props of a Message component?